2018-09-10-was-ist-das-xcamp-und-wer-sollte-teilnehmen.md

```diff
@@ -0,0 +1,18 @@
+---
+title:     Video-Interview: Was ist das XCamp und wer sollte teilnehmen?
+author:    Jens Bothmer
+published: 10.09.2019
+image:     
```

Comment:
We need the preview image from the old blog list here:
https://xcamp.autentity.net/agile-barcamp-blog/page/2/

-> https://xcamp.autentity.net/wp-content/uploads/sites/18/2018/06/Blog_Kachel-700x441.png

![](https://xcamp.autentity.net/wp-content/uploads/sites/18/2018/06/Blog_Kachel-700x441.png)


![](https://xcamp.autentity.net/wp-content/uploads/sites/18/2018/06/Blog_Kachel-700x441.png)

Reply:
So I was in wrong webpage! Sorry!

Reply:
I found the image and uploaded it to this brance. Images seem to have certificate problems in autentity website, but I could per drag and drop copy the images for my desktop. Direkt download didn't work.

Reply:
Yes, the certificate problem was the reason to move to my server and to move to this Markdown solution instead of WordPress.